LICENCE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+Copyright (C) 2015 Crown Copyright (Government Digital Service)
```

Comment:
Any reason the date is 2015?

Reply:
First commit was October 2015. We could put a range but we don't in our other licences.